docs/2.0/upgrade.md

```diff
@@ -4,6 +4,13 @@ We generally push changes behind the scenes, so you don't have to update your co
 
 Follow these guides to make sure your configuration files are up to date.
 
+## Upgrade from 2.29 to 2.30
+
+We have discovered that assigning different weights to two methods that police the same thing can be a complex and challenging task. As a result `upload_attachments?`, `download_attachments?`, and `delete_attachments?` become obsolete.
+You can safely delete it from all policies.
```

Comment:
```suggestion
You can safely delete it from all policies.

You may now introduce field-level authorization methods such as `upload_avatar?`, `download_avatar?`, and `delete_avatar?` if you have an `avatar` `file` field.
```

Reply:
We told them what to remove, we should give them a TL;DR; version of what to do next before sending them to the big docs.